### DISCUSSION TITLE
Welcome to homebridge-eufy-security Discussions!

### DISCUSSION THREAD

**Discussion by samemory** [samemory/homebridge-eufy-security-control#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andyrooo82** [comment]
Thanks @samemory @lenoxys @schliemann for making such a great plugin. It's working great so far thanks to your time and efforts. Can't wait to see this grow and develop.

I do have one question. On your roadmap, you mention you're going to add doorbell support. Is that the notification support that was added a couple of days ago or does this mean full doorbell support e.g. press, motion and camera?

Thanks again, appreciate your hard work!

**Reply by samemory** [reply]
Gald to hear it's working for you! 
I would like to add more support for the doorbell eventually. However, I currently do not have one so I'll be relying on the other contributors for testing out doorbell features. Given this is just a side project, no real estimate or promise it will ever happen though.

**Comment by PeterAlexander75** [comment]
I’d be happy to contribute and test for you. 

Also, this plugin for the doorbell does work, if that’s if any use. 

https://github.com/birkir/homebridge-plugin-eufy-security

**Comment by andyrooo82** [comment]
I'd also be happy to test for you

**Reply by samemory** [reply]
I have started to work on this. I have the doorbell code in a separate branch right now. Are you able to deploy it locally on your homebridge to test? It's not published to NPM.
https://github.com/samemory/homebridge-eufy-security/tree/doorbell
